### ISSUE TITLE
Default TypeScript skip expression for TypeScript files does not respect multiple reference elements

### ISSUE THREAD

**Issue by seangifford** [rubicon-oss/LicenseHeaderManager#102]
The default TypeScript skip expression for TypeScript files does not respect multiple <reference/> elements.

For example, if I have:
```
/// <reference path="file1.ts" />
/// <reference path="file2.ts" />
/// <reference path="file2.ts" />
```

After inserting the file header, it will look like this:
```
/// <reference path="file1.ts" />
/**
 * Some file header
 */
/// <reference path="file2.ts" />
/// <reference path="file2.ts" />
```

We fixed this by changing the skip expression to:
 (/// \*<reference.\*/>( |\t)\*(\n|\r\n|\r)?)\*

**Comment by FlorianDecker** [comment]
Thank you very much for raising this issue! We will fix this in the next release.

**Comment by FlorianDecker** [comment]
Fixed in efa7d515df677cde3d54f303c3c2c16843b28f70
